eventually/authentication/models.py

```diff
@@ -0,0 +1,161 @@
+
+    def get_email(self):
+        """
+        :return: email of the user.
+        """
+        return self.email
+
+    def get_updated_at(self):
+        """
+        :return: returns last update date in form of a timestamp
+        """
+        date = format(self.updated_at, u'U')
+        return date
+
+    def get_created_at(self):
+        """
+        :return: creation date of a user record in form of a timestamp
+        """
+        return format(self.created_at, u'U')
+
+    #'FIXME: which of these to remove? to_dict or myattr_to_dict ?'
+    def to_dict(self):
+        """
+        :return: Returns an object in form of a dictionary. Methods inherited from the parent object are also shown.
+        """
+        return self.__dict__
+
+    def myattr_to_dict(self):
+        """
+        :return: manually-defined attributes of the object. Inherited attributes aren't shown.
+        """
+        return {
+                'id': self.id,
+                'first_name': self.first_name,
+                'middle_name': self.middle_name,
+                'last_name': self.last_name,
+                'email': self.email,
+                'password': self.password,
+                'created_at': format(self.created_at, u'U'),
+                'updated_at': format(self.updated_at, u'U')
+                }
+
+    def update(self, **kwargs):
+        """
+        Updates user profile in the database with the specified parameters. \n
+        :param kwargs: takes a list of parameters (1-5) to be changed.
+        :return: returns True if object has been updated or False if the number of parameters was incorrect
+        """
+        if len(kwargs) == 0:
+            return False
+        if len(kwargs) > 5:
+            return False
+        else:
+            user = CustomUser.get_by_id(self.id)
+            if 'first_name' in kwargs: user.first_name = kwargs['first_name']
+            if 'middle_name' in kwargs: user.middle_name = kwargs['middle_name']
+            if 'last_name' in kwargs: user.last_name = kwargs['last_name']
+            if 'email' in kwargs: user.email = kwargs['email']
+            if 'password' in kwargs: user.password = kwargs['password']
+            user.updated_at = datetime.now()
+            user.save()
+            return True
+
+    #'FIXME: Remove the method below at the end of the development'
+    @staticmethod
+    def make(number_of_users):
+        """
+        :param number_of_users: number of user to create
+        :return: nothing. Users will be created in the db.
+        """
+        for user_number in range(number_of_users):
+            first_name = str(user_number) + '_first_name'
+            second_name = str(user_number) + '_second_name'
+            middle_name = str(user_number) + '_middle_name'
+            email = str(user_number)+'yay@co.uk'
+            password = user_number*2
+            CustomUser.create(first_name, second_name, middle_name, email, password).save()
+        pass
```

Comment:
use pass for what?

Reply:
Instead of the return statement. Though not needed here, removed it.